modules/appeals_api/spec/docs/appeals_status/v1_spec.rb

```diff
@@ -10,101 +10,115 @@ def swagger_doc
   "modules/appeals_api/app/swagger/appeals_status/v1/swagger#{DocHelpers.doc_suffix}.json"
 end
 
-# rubocop:disable RSpec/VariableName, Layout/LineLength, RSpec/RepeatedExample
+# rubocop:disable RSpec/VariableName, Layout/LineLength
 describe 'Appeals Status', swagger_doc:, type: :request do
   include DocHelpers
   let(:Authorization) { 'Bearer TEST_TOKEN' }
 
   path '/appeals' do
     get 'Retrieve appeals status for the Veteran with the supplied ICN' do
-      scopes = AppealsApi::V1::AppealsController::OAUTH_SCOPES[:GET]
       tags 'Appeals Status'
       operationId 'getAppealStatus'
 
       description "Returns a list of all known appeal records for the given Veteran. Includes details of each appeal's current status, priority, and history of updates."
 
-      security DocHelpers.oauth_security_config(scopes)
+      security DocHelpers.oauth_security_config(AppealsApi::V1::AppealsController::OAUTH_SCOPES[:GET])
       consumes 'application/json'
       produces 'application/json'
 
-      example_va_user = 'va.api.user+idme.025@gmail.com'
-
       parameter(
         parameter_from_schema('shared/v0/icn.json', 'properties', 'icn').merge(
           {
             name: :icn,
             in: :query,
-            required: true
+            description: "Veteran's Master Person Index (MPI) Integration Control Number (ICN). Optional when using a veteran-scoped token. Required when using a representative- or system-scoped token.",
```

Comment:
This string is the main place where I was able to make it known in the docs _where_ the `icn` parameter is required - I added examples that illustrate this, but I'm not sure how much more I can do with these examples. I do think we could look at updating the API description (working with UX) to help us make the behavior clear. I can try that in another PR.